tasks/services/plans.js

```diff
@@ -42,11 +42,12 @@ function createChildTaskPlans (hook) {
   const childTaskRecipes = taskRecipe.childTaskRecipes
   return Promise.all(
     childTaskRecipes.map((childTaskRecipe) => {
+      var params = hook.params[childTaskRecipe.id]
       return taskPlans.create({
         parentTaskPlanId: hook.result.id,
         assigneeId: hook.data.assigneeId,
         taskRecipeId: childTaskRecipe.id,
-        params: hook.data.params
+        params: JSON.stringify(params)
```

Comment:
this should already be done as a before hook on the service, `encodeParams` in https://github.com/root-systems/cobuy/blob/master/tasks/services/plans.js#L21-L32.